.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.18.x
+          go-version: "1.21"
```

Comment:
could you make it same as `.github/workflows/test.yaml`?
```yaml
      - uses: actions/checkout@v4
      - uses: actions/setup-go@v5
        with:
          go-version-file: "go.mod"
```

Reply:
fixed